app/services/AppealService.scala

```diff
@@ -90,12 +89,12 @@ class AppealService @Inject()(penaltiesConnector: PenaltiesConnector,
     val enrolmentKey = constructMTDVATEnrolmentKey(userRequest.vrn)
     val appealType = userRequest.session.get(SessionKeys.appealType)
     val isLPP = appealType.contains(PenaltyTypeEnum.Late_Payment.toString) || appealType.contains(PenaltyTypeEnum.Additional.toString)
-    val agentDetails = getAgentDetails
+    val agentReferenceNo = userRequest.arn.getOrElse("")
```

Comment:
You don't need to do the getOrElse here
```suggestion
    val agentReferenceNo = userRequest.arn
```
The above is fine ^